SalesforcePy/sfdc.py

```diff
@@ -46,6 +46,19 @@
 
 --boundary_string--"""
 
+class ApexRest(commons.BaseRequest):
+    """Performs a request to `'/services/apexrest/<action>'`"""
+
+    def __init__(
+            self, session_id, instance_url, action, http_method, request_params, request_body, **kwargs):
+        if request_params is not None:
+            p = urlencode(request_params)
+            action = '%s?%s' % (action, p)
+
+        super().__init__(
+                session_id, instance_url, http_method=http_method, request_body=request_body, **kwargs)
+        self.service = f"/services/apexrest/{action}"
```

Comment:
FYI [f-strings](https://realpython.com/python-f-strings/) where introduced in Python 3.6, so the tests running with tox and Python 3.5 will still fail. Should be fine if #68 is also done as part of the next release. Otherwise you might want to change this to:
```suggestion
        self.service = '/services/apexrest/%s' % action
```
Sorry I left the f-string there as part of my comment in #64.